src/uwtools/tests/drivers/test_chgres_cube.py

```diff
@@ -140,6 +142,12 @@ def test_ChgresCube_namelist_file_missing_base_file(caplog, driverobj):
     assert regex_logged(caplog, "missing.nml: State: Not Ready (external asset)")
 
 
+def test_ChgresCube_output(driverobj):
+    with raises(UWNotImplementedError) as e:
+        assert driverobj.output
+    assert str(e.value) == "The output() method is not yet implemented for this driver"
```

Comment:
A similar test is added to each driver test module. When a driver eventually provides a custom implementation, its test will need to be updated to test the new, accurate behavior.